### PR TITLE
Brings back sane wires coloring

### PR DESCRIPTION
**Pull request by Core0verload**
Amount of wire colors was brought down to 24, from insane _117_. Airlocks have 16 wires in them, and, honestly, if you need more than 20 wires in a single device, you are doing it wrong.

"lightgoldenrodyellow", "blanchedalmond", "aliceblue", "antiquewhite", "chartreuse", and  such aren't even real colors.

### CODE REVIEW

**Comment by bgobandit** [comment]
good stuff benjamin quesnel

**Comment by PKPenguin321** [comment]
>93 lines removed

wew

**Comment by Iamgoofball** [comment]
fuck off :-1:

**Comment by Iamgoofball** [comment]
Those colors are real colors, by the way. Good job lying in your PR description.
http://www.color-hex.com/color/7fff00 charteruse
http://www.color-hex.com/color/fafad2 light goldenrod yellow
chrome_2016-03-27_16-52-52

**Comment by ChangelingRain** [comment]
I like it, it's funny.

**Comment by PKPenguin321** [comment]
@Iamgoofball those colors are ugly as fuck

**Comment by lzimann** [comment]
no real point in having a shitload of colors tbh

File: code/datums/wires/wires.dm [hunk 1]
```diff
@@ -1,119 +1,26 @@
 var/list/wire_colors = list( // http://www.crockford.com/wrrrld/color.html
-	"aliceblue",
-	"antiquewhite",
-	"aqua",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 2]
```diff
@@ -1,119 +1,26 @@
 var/list/wire_colors = list( // http://www.crockford.com/wrrrld/color.html
-	"aliceblue",
-	"antiquewhite",
-	"aqua",
-	"aquamarine",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 3]
```diff
@@ -1,119 +1,26 @@
 var/list/wire_colors = list( // http://www.crockford.com/wrrrld/color.html
-	"aliceblue",
-	"antiquewhite",
-	"aqua",
-	"aquamarine",
-	"beige",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 4]
```diff
@@ -1,119 +1,26 @@
...
-	"aquamarine",
-	"beige",
-	"blanchedalmond",
 	"blue",
-	"blueviolet",
 	"brown",
-	"burlywood",
-	"cadetblue",
-	"chartreuse",
-	"chocolate",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 5]
```diff
@@ -1,119 +1,26 @@
...
-	"coral",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 6]
```diff
@@ -1,119 +1,26 @@
...
-	"deepskyblue",
-	"dimgray",
-	"dodgerblue",
-	"firebrick",
-	"floralwhite",
-	"forestgreen",
-	"fuchsia",
-	"gainsboro",
-	"ghostwhite",
-	"gold",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 7]
```diff
@@ -1,119 +1,26 @@
...
-	"goldenrod",
 	"gray",
 	"green",
 	"greenyellow",
-	"honeydew",
-	"hotpink",
-	"indianred",
-	"ivory",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 8]
```diff
@@ -1,119 +1,26 @@
...
-	"khaki",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 9]
```diff
@@ -1,119 +1,26 @@
...
-	"lavender",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 10]
```diff
@@ -1,119 +1,26 @@
...
 	"crimson",
 	"cyan",
-	"deeppink",
-	"deepskyblue",
-	"dimgray",
-	"dodgerblue",
-	"firebrick",
-	"floralwhite",
-	"forestgreen",
-	"fuchsia",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 11]
```diff
@@ -1,119 +1,26 @@
...
-	"lightseagreen",
-	"lightskyblue",
-	"lightslategray",
-	"lightsteelblue",
 	"lightyellow",
 	"lime",
 	"limegreen",
-	"linen",
 	"magenta",
-	"maroon",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 12]
```diff
@@ -1,119 +1,26 @@
...
-	"mintcream",
-	"mistyrose",
-	"moccasin",
-	"navajowhite",
-	"oldlace",
 	"olive",
-	"olivedrab",
 	"orange",
 	"orangered",
-	"orchid",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 13]
```diff
@@ -1,119 +1,26 @@
...
-	"palegoldenrod",
-	"palegreen",
-	"paleturquoise",
-	"palevioletred",
-	"papayawhip",
-	"peachpuff",
-	"peru",
 	"pink",
-	"plum",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 14]
```diff
@@ -1,119 +1,26 @@
...
-	"slateblue",
-	"slategray",
-	"snow",
-	"springgreen",
-	"steelblue",
-	"tan",
-	"teal",
-	"thistle",
-	"tomato",
-	"turquoise",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 15]
```diff
@@ -1,119 +1,26 @@
...
-	"sienna",
-	"silver",
-	"skyblue",
-	"slateblue",
-	"slategray",
-	"snow",
-	"springgreen",
-	"steelblue",
-	"tan",
-	"teal",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 16]
```diff
@@ -1,119 +1,26 @@
...
-	"seashell",
-	"sienna",
-	"silver",
-	"skyblue",
-	"slateblue",
-	"slategray",
-	"snow",
-	"springgreen",
-	"steelblue",
-	"tan",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 17]
```diff
@@ -1,119 +1,26 @@
...
-	"salmon",
-	"sandybrown",
-	"seagreen",
-	"seashell",
-	"sienna",
-	"silver",
-	"skyblue",
-	"slateblue",
-	"slategray",
-	"snow",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 18]
```diff
@@ -1,119 +1,26 @@
...
 	"red",
-	"rosybrown",
-	"royalblue",
-	"saddlebrown",
-	"salmon",
-	"sandybrown",
-	"seagreen",
-	"seashell",
-	"sienna",
-	"silver",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 19]
```diff
@@ -1,119 +1,26 @@
...
 	"purple",
 	"red",
-	"rosybrown",
-	"royalblue",
-	"saddlebrown",
-	"salmon",
-	"sandybrown",
-	"seagreen",
-	"seashell",
-	"sienna",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

File: code/datums/wires/wires.dm [hunk 20]
```diff
@@ -1,119 +1,26 @@
...
-	"peru",
 	"pink",
-	"plum",
-	"powderblue",
 	"purple",
 	"red",
-	"rosybrown",
-	"royalblue",
-	"saddlebrown",
-	"salmon",
```
Comment (ChangelingRain) [inline code/datums/wires/wires.dm]: keep

**Comment by Core0verload** [comment]
The algorithm that I used to remove colors is simple. "If you can't imagine a wire insulation with that color in 5 seconds, down it goes." That's why "gold" and "silver" wires are removed - there is no such thing as gold-colored wire insulation.

**Comment by optimumtact** [comment]
Yeah seems sensible, never did understand nt's fascination with that many wire insulation colours when the main lines are only red/green/yellow/blue

File: code/datums/wires/wires.dm (hunk 17 above)
Comment (KazeEspada) [inline code/datums/wires/wires.dm]: meh? Snow is white. TBH.

File: code/datums/wires/wires.dm (hunk 19 above)
Comment (KazeEspada) [inline code/datums/wires/wires.dm]: Thats not a color thats a car.

File: code/datums/wires/wires.dm (hunk 20 above)
Comment (KazeEspada) [inline code/datums/wires/wires.dm]: Thats a fish.

**Comment by Core0verload** [comment]
Makes me wonder: is there any argument for having a ton of unpronounceable unreadable colors?

**Comment by octareenroon91** [comment]
I think @neersighted did it because he could, as part of putting hacking on tgui.

The colors are web standard, so the name string works for both display and markup.

It's a lot of colors which might not seem relevant, but there are real-life wire sets we can't model, such as the color/color-stripes-on-white pairs in ethernet cable.

I am not sqggesting striped wires in ss13

**Comment by Core0verload** [comment]
Striped wires are fine, if we would ever run out of those 24 colors.

File: code/datums/wires/wires.dm (hunk 20 above)
Comment (Iamgoofball) [inline code/datums/wires/wires.dm]: its also a color

File: code/datums/wires/wires.dm (hunk 19 above)
Comment (Iamgoofball) [inline code/datums/wires/wires.dm]: its a color too

**Comment by neersighted** [comment]
Meh, I did this because I needed more than the 12 colors that existed at the time and trying to find an A E S T H E T I C set of colors was too hard, so I added all of them. Then we eliminated british colors, so here we are... Do as you will...

**Comment by Cheridan** [comment]
>here is no such thing as gold-colored wire insulation. 

Gold and Silver don't have to be metallic colors: 
https://s.graphiq.com/sites/default/files/2307/media/images/Gold_Metallic_1395343.png
https://s.graphiq.com/sites/default/files/2307/media/images/Silver_429807_i0.png

**Comment by optimumtact** [comment]
Why are we bikeshedding about this it's such a fine trivial change for QOL just merge it already please

**Comment by Core0verload** [comment]
>Gold and Silver don't have to be metallic colors:
https://s.graphiq.com/sites/default/files/2307/media/images/Gold_Metallic_1395343.png
https://s.graphiq.com/sites/default/files/2307/media/images/Silver_429807_i0.png

Without the metallic it's just gray and dirty yellow. And wires are usually a bit dirty.

File: code/datums/wires/wires.dm (hunk 5 above)
Comment (lordpidey) [inline code/datums/wires/wires.dm]: coral?  Doesn't coral come in all colors?

**Comment by Cheridan** [comment]
It's gray with a slightly bluish tint.

Also @optimumtact the bike shed should be painted bleached almond and be at least twice as large.

**Comment by optimumtact** [comment]
PLEASE CHERIDAN PLEASE RELEASE ME

**Comment by demonfiren** [comment]
Actually, chartreuse is a real colour.

It's just no colour anyone would ever want to see.

**Comment by Cheridan** [comment]
chartreuse did nothing wrong

https://s.graphiq.com/sites/default/files/2307/media/images/Chartreuse_429758_i0.png

**Comment by Cheridan** [comment]
sorry @optimumtact @Core0verload the downvotes have spoken

**Comment by Core0verload** [comment]
Yet I don't understand why keeping 100+ wires with weird names and missing spacebars is better than having 20 simple and common wire colors. Goof's "fuck off :-1:" is the best argument I saw in this discussion, and that says a lot.

**Comment by Iamgoofball** [comment]
<3

**Pull request closed without merging**